parrot/src/irods_reli.cc

```diff
@@ -117,6 +120,17 @@ static struct irods_server * connect_to_host( const char *hostport )
 		return 0;
 	}
 
+	if(!got_irods_passwd_file) {
+		std::string passwd_file(getenv("HOME"));
```

Comment:
I think we need to handle HOME not being set. IIRC, in Condor, if you run Parrot directly without a shell, you don't get HOME.

I looked at iRODS's source and it appears they just concatenate the two though, without checking and no fallback. We should check though.